Src/Utils/INI.h

```diff
@@ -28,6 +28,8 @@ class INIFile {
         INIFile(const PGE::FilePath& filename);
         ~INIFile();
 
+        INIFile* iniFileFactory(const PGE::FilePath& filename);
+        void close(void* ptr);
```

Comment:
This belongs in INIDefinitions, if anywhere

Reply:
I have moved this to INIDefinitions.